app/src/main/res/values/strings.xml

```diff
@@ -136,5 +136,6 @@
     <!--settings profile dialog-->
     <string name="message">Are you sure you want to log out?</string>
     <string name="cancel">Cancel</string>
+    <string name="password_constraints">* Atleast 6 characters</string>
```

Comment:
where is this used?

Reply:
otherwise LGTM 👍

Reply:
Now it is not used! I'll remove this today itself!